build-ci-wheels/action.yml

```diff
@@ -1,57 +1,59 @@
-name: "Build wheels for a C-extension Python library."
-description: "Build wheels for a C-extension Python library."
+name: >
+  Build CI wheels
+
+description: >
+  Build wheels for a Python library. This action is expected to be used within a
+  matrix job. Its goal is to generate wheel files for every combination of
+  operating system, build system and Python version.
```

Comment:
```suggestion
  operating system, build system, and Python version.
```

Reply:
Agh, sorry for missing this coma, Kathy.